### PR TITLE
Credits for version 8.7

### PR DESCRIPTION
**Pull request by mattam82**
Here's a first draft of the credits for version 8.7, please correct any mistakes or missing bits, I mostly worked from the current CHANGES file and may have missed important bits. In particular, @ejgallego I wonder what I should say about the XML protocol and STM work.

### CODE REVIEW

File: doc/refman/RefMan-pre.tex [hunk 1]
```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
...
+Pierre Letouzey, Assia Mahboubi, Cyprien Mangin, Erik Martin-Dorel,
+Olivier Marty, Guillaume Melquiond, Sam Pablo Kuper, Benjamin Pierce,
+Pierre-Marie Pédrot, Lars Rasmusson, Valentin Robert, Yann Régis-Gianas,
+Thomas Sibut-Pinote, Michael Soegtrop, Matthieu Sozeau, Arnaud Spiwack,
+Paul Steckler, George Stelle, Pierre-Yves Strub, Enrico Tassi, Hendrik
+Tews, Amin Timany, Laurent Théry, Vadim Zaliva and Théo Zimmermann.
+
+The development process was coordinated by Matthieu Sozeau with the help
+of Maxime Dénès, who was also in charge of the release process. Théo
+Zimmermann endorsed the role of maintainer for this release.
```
Comment (gares) [inline doc/refman/RefMan-pre.tex]: Why the past tense for Theo?

Comment (mattam82) [inline doc/refman/RefMan-pre.tex]: Hmm, right, I'll make it present

**Comment by ejgallego** [comment]
@mattam82 IMHO not worth saying anything, we did many minor cleanups but the large changes have been all postponed to 8.8.

File: doc/refman/RefMan-pre.tex [hunk 2]
```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
...
+Version 8.7 also comes with a bunch of smaller-scale changes and improvements
+regarding the different components of the system. We shall only list a
+few of them.
+
+The efficiency of the whole system has been significantly improved
+thanks to contributions from Pierre-Marie Pédrot, Maxime Dénès and
+Matthieu Sozeau and performance issue tracking by Jason Gross and Paul
+Steckler.
+
+Hugo Herbelin added support for side effects hooks in \texttt{cbv},
```
Comment (herbelin) [inline doc/refman/RefMan-pre.tex]: Thomas Sibut-Pinute should be mentioned I guess. (And the plugin be released.)

File: doc/refman/RefMan-pre.tex [hunk 3]
```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
...
+contributions. Code reviews and continuous integration testing were
+systematically used before integration of new features, with an
+important focus given to compatibility and performance issues, resulting
+in a hopefully more robust release than {\Coq} 8.6 while maintaining
+compatibility.
+
+Coq Enhancement Proposals (CEPs for short) and open pull-requests
+discussions were used to discuss publicly the new features.
+
+The {\Coq} consortium, an organisation directed towards users and
```
Comment (herbelin) [inline doc/refman/RefMan-pre.tex]: To continue with the US spelling rule, should be "organization" I think.

File: doc/refman/RefMan-pre.tex [hunk 4]
```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
...
+Maxime Dénès for MacOS X. Packages are regularly built on the
+Travis continuous integration server.
+
+The contributors for this version are Abhishek Anand, C.J. Bell, Yves
+Bertot, Frédéric Besson, Tej Chajed, Pierre Courtieu, Maxime Dénès,
+Julien Forest, Gaëtan Gilbert, Jason Gross, Hugo Herbelin, Emilio Jesús
+Gallego Arias, Ralf Jung, Matej Košík, William Lawvere, Xavier Leroy,
+Pierre Letouzey, Assia Mahboubi, Cyprien Mangin, Erik Martin-Dorel,
+Olivier Marty, Guillaume Melquiond, Sam Pablo Kuper, Benjamin Pierce,
+Pierre-Marie Pédrot, Lars Rasmusson, Valentin Robert, Yann Régis-Gianas,
```
Comment (herbelin) [inline doc/refman/RefMan-pre.tex]: Maybe Lionel Rieg should be mentioned for his work on notations, or maybe did Emilio redo everything?

File: doc/refman/RefMan-pre.tex [hunk 5]
```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
...
+
+Packaging tools and software development kits were prepared by Michael
+Soegtrop with the help of Maxime Dénès and Enrico Tassi for Windows, and
+Maxime Dénès for MacOS X. Packages are regularly built on the
+Travis continuous integration server.
+
+The contributors for this version are Abhishek Anand, C.J. Bell, Yves
+Bertot, Frédéric Besson, Tej Chajed, Pierre Courtieu, Maxime Dénès,
+Julien Forest, Gaëtan Gilbert, Jason Gross, Hugo Herbelin, Emilio Jesús
+Gallego Arias, Ralf Jung, Matej Košík, William Lawvere, Xavier Leroy,
```
Comment (herbelin) [inline doc/refman/RefMan-pre.tex]: Are we sure that William Lawvere is the real name of the contributor using this name? (Not that pseudonyms are bad, but the collision with a well-known mathematician name might be worth to be clarified.)

**Review by herbelin — COMMENTED** [review]
Well done. Congratulations.

File: doc/refman/RefMan-pre.tex (hunk 5 above)
Comment (mattam82) [inline doc/refman/RefMan-pre.tex]: I had a doubt, thinking they just shared their last names. All the information I have is his profile here: https://github.com/william-lawvere I can add a "(pseudonym)" mention?

File: doc/refman/RefMan-pre.tex (hunk 2 above)
Comment (mattam82) [inline doc/refman/RefMan-pre.tex]: Done

File: doc/refman/RefMan-pre.tex (hunk 4 above)
Comment (mattam82) [inline doc/refman/RefMan-pre.tex]: Right.

File: doc/refman/RefMan-pre.tex (hunk 3 above)
Comment (mattam82) [inline doc/refman/RefMan-pre.tex]: Indeed

**Comment by mattam82** [comment]
Integrated Hugo's comments and run a spell-checker as well

File: doc/refman/RefMan-pre.tex (hunk 5 above)
Comment (Zimmi48) [inline doc/refman/RefMan-pre.tex]: The best way to know whether it is a real name or a pseudonym would be to ask @william-lawvere directly...

Comment (mattam82) [inline doc/refman/RefMan-pre.tex]: I sent him an email, let's see.

File: doc/refman/RefMan-pre.tex [hunk 6]
```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
...
+\begin{itemize}
+\item New tactics: variants of tactics supporting existential variables
+  \texttt{eassert}, \texttt{eenough}, etc... by Hugo Herbelin. Tactics
+  \texttt{extensionality in H} and \texttt{inversion\_sigma} by Jason
+  Gross, \texttt{specialize with ...} accepting partial bindings by
+  Pierre Courtieu.
+\item Cumulative Polymorphic Inductive Types, allowing cumulativity of
+  universes to go through applied inductive types, by Amin Timany and
+  Matthieu Sozeau.
+\item Integration of the \texttt{Ssreflect} plugin and its documentation in the
```
Comment (maximedenes) [inline doc/refman/RefMan-pre.tex]: I believe the official case is SSReflect

Comment (Zimmi48) [reply doc/refman/RefMan-pre.tex]: It would help if it were used consistently by SSReflect developers ^^

Comment (maximedenes) [reply doc/refman/RefMan-pre.tex]: indeed :)

Comment (Zimmi48) [reply doc/refman/RefMan-pre.tex]: @mattam82 ?

Comment (mattam82) [reply doc/refman/RefMan-pre.tex]: done

**Comment by maximedenes** [comment]
Thanks, let's merge!

**Pull request merged**